client/src/js/components/bhEmployeeFinder.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * The employee finder component
+ *
+ * @module components/bhEmployeeFinder
+ *
+ * @description a component to deal with employee, it let you choose a employee
+ * 
```

Comment:
It would be nice to have more of a description.  How do you use it?  What API does the component expose?

We are building components to share between modules, it would be nice to have some documentation associated with them as we develop each new component.